features/template/test_template/success.feature

```diff
@@ -0,0 +1,27 @@
+Feature: test templates
+
+  When developing an exosphere service template
+	I want to be able to test it to make sure it can be used by exocom
+	So that I don't have broken templates
+
+  - run "exo template test" in the directory of an exosphere template to test
+    adding it to an application and run the tests
+
+  - Structure rules for a valid template
+    - contains "project.json"
+    - contains "template" folder
+      - contains a single folder
+        - contains "service.yml"
+        - contains "Dockerfile"
+        - contains "tests/Dockerfile"
```

Comment:
These structure rules were written by @charlierudolph. Yes, these 3 lines should be indented to show the structure of the template directory because the files live in the single folder in `'template'`.